dialoguekit/core/dialogue.py

```diff
@@ -77,6 +79,11 @@ def utterances(self) -> List[Utterance]:
         """Returns the utterances in the dialogue."""
         return self._utterances
 
+    @property
+    def utterance_feedbacks(self) -> List[UtteranceFeedback]:
+        """Returns feedbacks on utterances in the dialogue."""
+        return self._utterance_feedbacks
+
```

Comment:
Is this ever going to be accessed this way? I'd assume that we either want to fetch the feedback for a given utterance (based on utterance_id) or get the entire dialogue with all utterances and feedbacks.

Reply:
Fixed